### PR TITLE
[ SW-501 ] fix: Update privacy policy

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Updating the privacy policy according to [this request](https://5afe.slack.com/archives/C045P36MXQC/p1730108275091469)

## How this PR fixes it

- Adds markdown support for privacy policy

## How to test it

1. Open the privacy page
2. Observe the updates from the document

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/pages/privacy.tsx [hunk 1]
```diff
@@ -1,1004 +1,30 @@
...
+  React.PropsWithChildren<Omit<MUILinkProps, 'href'> & Pick<NextLinkProps, 'href' | 'as'>>
+> = ({ href = '', as, children, ...other }) => {
+  const isExternal = href.toString().startsWith('http')
+  return (
+    <NextLink href={href} as={as} passHref legacyBehavior>
+      <MUILink target={isExternal ? '_blank' : ''} rel="noreferrer" {...other}>
+        {children}
+      </MUILink>
+    </NextLink>
+  )
```
Comment (compojoom) [inline src/pages/privacy.tsx:21]: How about we move this to util function as we have the same function in the terms.tsx page?

File: src/markdown/privacy/privacy.md.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export { default } from '*.md'
+export const metadata = {
+  version: string,
+  last_update_date: string,
+}
```
Comment (compojoom) [inline src/markdown/privacy/privacy.md.d.ts:5]: You are specifying a frontmatter with this, but your md file doesn't have one. Look at terms.md

```
---
version: 1.2
last_update_date: September, 2024
---
```

Comment (usame-algan) [reply src/markdown/privacy/privacy.md.d.ts:5]: Afaik we are not using those anywhere in the app like we do for terms so I will remove the `.d.ts` file

File: src/pages/privacy.tsx (hunk 1 above)
Comment (usame-algan) [reply src/pages/privacy.tsx:21]: done

**Review by compojoom — APPROVED** [review]
👍

**Comment by liliya-soroka** [comment]
The formatting is broken in a new version. Please check why it looks so different and sometimes with the wrong numbering

**Comment by usame-algan** [comment]
> The formatting is broken in a new version. Please check why it looks so different and sometimes with the wrong numbering

Updated the document. The numbering should be fixed. The styles might look a bit different than before but it is more consistent with the styling of the terms page now.

**Comment by francovenica** [comment]
I was able to see that the changes are there, but I don't have access to the doc, so I cannot confirm everythign is correct.

I requested access and I'll wait to have it

**Comment by liliya-soroka** [comment]
verified by the product - mhttps://www.notion.so/safe-global/fix-Update-privacy-policy-1378180fe573801a9df1d33eb19d6e4c?pvs=4

**Pull request merged**